### PR TITLE
add lang clarification to translatable attributes section.

### LINKED ISSUE #1365: Attribute with natural language but no language metadata
spinbutton role
https://raw.githack.com/w3c/aria/2020-09_CR/index.html#spinbutton

> Supported States and Properties: [...] aria-valuetext

[‘Aria-valuetext’](https://raw.githack.com/w3c/aria/2020-09_CR/index.html#aria-valuetext) is an attribute that contains natural-language text, with no way to explicitly set the language. (Writing direction is probably less of a problem, as the content is usually spoken, not displayed.)

The language is probably expected to be inherited from the element. If so, it would be good to mention it. It may be different from the language of the document. 

The attribute is not new. It existed already in ARIA 1.1. The only thing new is that in ARIA 1.2 it also applies to elements with role=spinbutton. So this issue probably applies to all occurrences of aria-valuetext and, in fact, to other aria-* attributes that contain text, too.

### PR DESCRIPTION
**Pull request by cookiecrook**
Resolves #1365

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1690.html" title="Last updated on Mar 8, 2022, 5:55 AM UTC (1b45be1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1690/157d711...1b45be1.html" title="Last updated on Mar 8, 2022, 5:55 AM UTC (1b45be1)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10320,14 +10320,16 @@ <h4>Example Attribute Usage</h4>
...
+		<h2>Translatable Attributes</h2>
+		<p>The HTML specification states that other specifications can define <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a>. The source translation language of each value is defined by use of <a href="https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes">the <code>lang</code> attribute</a> on the element or its ancestor elements.</p>
+		<p>In order to be understandable by assistive technology users, the values of the following <a>states</a> and <a>properties</a> are <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a> and should be translated when a page is localized:</p>
 		<ul>
 			<li><pref>aria-label</pref></li>
 			<li><pref>aria-placeholder</pref></li>
 			<li><pref>aria-roledescription</pref></li>
 			<li><pref>aria-valuetext</pref></li>
 		</ul>
+
```
Comment (cookiecrook) [inline index.html:10332]:
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -10320,14 +10320,16 @@ <h4>Example Attribute Usage</h4>
...
+		<h2>Translatable Attributes</h2>
+		<p>The HTML specification states that other specifications can define <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a>. The source translation language of each value is defined by use of <a href="https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes">the <code>lang</code> attribute</a> on the element or its ancestor elements.</p>
+		<p>In order to be understandable by assistive technology users, the values of the following <a>states</a> and <a>properties</a> are <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a> and should be translated when a page is localized:</p>
```
Comment (cookiecrook) [inline index.html:10325]:
```suggestion
		<p>To be understandable by assistive technology users, the values of the following <a>states</a> and <a>properties</a> are <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a> and should be translated when a page is localized:</p>
```

File: index.html [hunk 3]
```diff
@@ -10320,8 +10320,9 @@ <h4>Example Attribute Usage</h4>
 		</section>
 	</section>
 	<section>
-		<h2>Translatable States and Properties</h2>
-		<p>The HTML specification states that other specifications can define <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a>. In order to be understandable by assistive technology users, the values of the following <a>states</a> and <a>properties</a> are <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a> and should be translated when a page is localized:</p>
+		<h2>Translatable Attributes</h2>
+		<p>The HTML specification states that other specifications can define <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a>. The source translation language of each value is defined by use of <a href="https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes">the <code>lang</code> attribute</a> on the element or its ancestor elements.</p>
```
Comment (carmacleod) [inline index.html:10324]: Maybe say "attribute value"? When it just says "value", I'm thinking, "what value?", "value of what?".

```suggestion
		<p>The HTML specification states that other specifications can define <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a>. The source translation language of each attribute value is defined by use of <a href="https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes">the <code>lang</code> attribute</a> on the element or its ancestor elements.</p>
```

**Review by carmacleod — APPROVED** [review]
+1 as-is, but might be a bit clearer if we insert the word "attribute" before "value".

**Review by MelSumner — APPROVED** [review]
Thank you for working on this. 👍

**Review by jnurthen — APPROVED** [review]

**Pull request merged**